### ISSUE TITLE
Token issues with unpublish command

### ISSUE THREAD

**Issue by u2berggeist** [Microsoft/vscode-vsce#269]
I can't enter in the Token for the unpublish command for whatever reason. For every charracter, it puts a `*` between then. Here's a screen shot from `cmd`:

![image](https://user-images.githubusercontent.com/20801821/41934074-b74d8834-7953-11e8-9bfe-0e9b330a02aa.png)

That came up after typing in `543234`. This also happens when I copy/paste a token as well. Below is an example of what happens there:
![image](https://user-images.githubusercontent.com/20801821/41934206-139642fc-7954-11e8-9858-4090d015797a.png)

I copy/pasted the token into the original prompt. When I hit backspace, it created the line below. On that second line, if I press another key, it also will show the letter I pressed along with a `*` right beside it. 

When I hit enter it gives `Error: Aborted` which says to me that it knows something's wrong, so it's not even checking the token.

I'm using version `1.43.0`

**Comment by joaomoreno** [comment]
If you paste an hit Enter, it doesn't work at all?

What if you type it? Does it work then?

**Comment by u2berggeist** [comment]
Correct, If I just paste and hit enter, it comes up with `Error: Aborted`.

Typing it comes up with the result in top screen capture. So there I typed `543234` and it put a `*` after every number I input. 

TBH, I haven't tried typing in the token because I didn't want to type in a 52 character token. I can do that if you'd like me to check though.
